src/charts/trends/trends-chart.html

```diff
@@ -1,7 +1,20 @@
-<div>
-  <span class="trend-header-pf" ng-if="config.title">{{config.title}}</span>
-  <span class="trend-title-big-pf">{{chartData.yData[chartData.yData.length-1]}}</span> <span class="trend-title-small-pf">{{config.units}}</span>
-  <div pf-sparkline-chart config="config" chart-data="chartData" chart-height="chartHeight"
-       show-x-axis="showXAxis" show-y-axis="showYAxis"></div>
-  <span class="trend-footer-pf" ng-if="config.timeFrame">{{config.timeFrame}}</span>
-</div>
+<span>
+  <div ng-class="{'trend-card-large-pf': showLargeCardLayout,'trend-card-small-pf': showSmallCardLayout}">
+
+    <span class="trend-header-pf" ng-if="config.title">{{config.title}}</span>
+
+    <span ng-if="showActualValue">
+      <span class="trend-title-big-pf">{{getLatestValue()}}</span>
+      <span class="trend-title-small-pf">{{config.units}}</span>
+    </span>
+    <span ng-if="showPercentageValue">
```

Comment:
You could use [`ng-switch`](https://docs.angularjs.org/api/ng/directive/ngSwitch) here instead, especially if we plan on adding additional modes.